Demo/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
 
     defaultConfig {
         applicationId "com.paypal.android"
-        minSdkVersion 21
+        minSdkVersion 23
```

Comment:
Not sure if we want to keep minSdkversion 21 and require min SDk level on the function that invokes the PayPal Checkout SDK. This would allow merchants who support 21 to use the PayPal module and only handle the case for API level 21 and 22.